packages/node-core/src/configure/NodeConfig.ts

```diff
@@ -182,7 +179,7 @@ export class NodeConfig<C extends IConfig = IConfig> implements IConfig {
       return this._config.dictionaryRegistry;
     }
 
-    return 'https://github.com/subquery/templates/raw/main/dictionary.json';
+    return 'https://github.com/subquery/templates/raw/main/dist/dictionary.json';
```

Comment:
Does this need to be done in the main sdk?

Reply:
It is same with current sdk, but in this branch haven't been synced, change here otherwise will break some tests